src/map/ai/helpers/pathfind.cpp

```diff
@@ -410,7 +414,7 @@ void CPathFind::StepTo(const position_t& pos, bool run)
         speed /= 2;
     }
 
-    float stepDistance = (speed / 10) / 2;
+    float stepDistance = (speed / 5) / 2;
```

Comment:
Whats this magic number? Could be defined at top of file (see zone_entities for examples of this, I think)

Reply:
This doesn't have a magic number yet. This needs to be extrapolated out into constant speed vector calculation vs doing it like this.